README.md

```diff
@@ -607,15 +607,15 @@ requests. This function is invoked by the middleware when a
 
 ### Built-in authorizers
 
-Redux Simple Auth currently ships with 1 authorizer. As this
+Redux Simple Auth currently ships with 2 authorizers. As this
 library matures, there are plans to implement more built-in authorizers. Refer
 to the [custom authorizers](#implementing-a-custom-authorizer) section to build
 your own.
 
 **OAuth2 Bearer**
 
 This authorizer is responsible for setting the `Authorization` header using the
-`Bearer` scheme.
+`Bearer` scheme. This authorizer expects an `access_token` field.
```

Comment:
I know I left this out, but should we link to the [RFC](https://tools.ietf.org/html/rfc6750#section-4) in regards to the structure of the returned token? Thoughts?

Here is the link: https://tools.ietf.org/html/rfc6750#section-4